### PR TITLE
Add basic English doc for install

### PR DESCRIPTION
**Pull request by yubozhao**
增加安装文档的英文翻译

### CODE REVIEW

File: docs/install_en.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+</style>
+
+<script>
+$('#latest-version').html(antdVersion.latest);
+
+if (antdVersion.stable) {
+  $('#stable-version').html(antdVersion.stable);
+} else {
+  $('#stable-version').html('暂无');
```
Comment (benjycui) [inline docs/install_en.md]: "暂无"或者可以译为"Comming soon" ?

**Comment by benjycui** [comment]
:+1:

File: docs/install_en.md (hunk 1 above)
Comment (yubozhao) [inline docs/install_en.md]: 可以。我回家后修改

Bo




On Thu, Dec 17, 2015 at 5:56 PM -0800, "Benjy Cui" <notifications@github.com> wrote:












In docs/install_en.md:
> +  color: #C05B4D;
> +  font-family: Consolas;
> +  margin-left: 0.3em;
> +  background: #FFF1E7;
> +  padding: 2px 5px;
> +  border-radius: 3px;
> +}
> +</style>
> +
> +<script>
> +$('#latest-version').html(antdVersion.latest);
> +
> +if (antdVersion.stable) {
> +  $('#stable-version').html(antdVersion.stable);
> +} else {
> +  $('#stable-version').html('暂无');



"暂无"或者可以译为"Comming soon" ?

**Comment by yubozhao** [comment]
@benjycui 改好了， 这个PR过了， 我再弄别的

**Comment by afc163** [comment]
:+1:

**Pull request merged**